Makefile

```diff
@@ -4,21 +4,27 @@ ifeq ($(ARCH),)
 ARCH=$(shell go env GOARCH)
 endif
 
+BUILD_META=-build$(shell date +%Y%m%d)
 ORG ?= rancher
 PKG ?= github.com/opencontainers/runc
 SRC ?= github.com/opencontainers/runc
-TAG ?= v1.0.0-rc92
+TAG ?= v1.0.0-rc92$(BUILD_META)
```

Comment:
Can we bump to the latest https://github.com/opencontainers/runc/releases/tag/v1.0.0-rc93